x-pack/plugins/event_log/server/plugin.test.ts

```diff
@@ -19,7 +20,9 @@ describe('event_log plugin', () => {
 
     const plugin = new Plugin(initializerContext);
     // serverless setup is currently empty, and there is no mock
```

Comment:
2 / 3 - Could you remove this comment since it's obviously no longer correct :-)